viz_scripts/mode_purpose_share.ipynb

```diff
@@ -25,166 +25,146 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "year = 2020\n",
-    "month = 11\n",
-    "program = \"prepilot\""
+    "year = 2021\n",
+    "month = 9\n",
+    "program = \"prepilot\"\n",
```

Comment:
we can revert these changes since they are not substantive.

Reply:
Removed this notebook